### DISCUSSION TITLE
PLEASE I REALLY NEED YOUR HELP TO RESTORE MY CODE

### DISCUSSION THREAD

**Discussion by lawbyte** [vercel/community#7348] [Help] [answered]
### Summary

Hi

please help me to restore my code, my project is still on there 
https://jekyll-theme-chirpy-git-imgbot-noxlaw.vercel.app/

but the repo is gone, please help me, i dont want to rewrite my blog
cz its really so long

please help me to recover

### Steps to Reproduce

Hi

please help me to restore my code, my project is still on there 
https://jekyll-theme-chirpy-git-imgbot-noxlaw.vercel.app/

but the repo is gone, please help me, i dont want to rewrite my blog
cz its really so long

please help me to recover

**Comment by pawlean** [comment]
Hi @lawbyte!

Sorry to hear this. I'm unsure how we can help here. Did you create a repository on GitHub and used Vercel to deploy it? How did you lose your code?

**Comment by lawbyte** [comment]
yeah thats true, i lost my code because im doing mass delete at fork repostory and forgot to exclude this repo

**Reply by pawlean** [reply, marked as answer]
Did you ever commit it onto GitHub? Or was it all stored locally?

We do not store the source code. However, you can manually download the output from the output tab. This can be easily accessed on the dashboard: navigate to `Dashboard > Project > Deployment > Source` tab. 

We do not offer a direct way to download everything at once; so you need to either use our API or manually copy/paste the pages you need: https://vercel.com/docs/rest-api/endpoints/deployments#list-deployment-files.

[a comment by an automated account is not shown]
